wdl2cwl/tests/test_cwl.py

```diff
@@ -1,137 +1,67 @@
+"""Tests for miniwdl."""
 import os.path
+from tempfile import NamedTemporaryFile
+from pathlib import Path
 import pytest
-import pathlib
-import filecmp
 
-from .. import main as wdl
+from ..main import main
 
 
 def get_file(path: str) -> str:
+    """Get file."""
     return os.path.join(os.path.dirname(__file__), path)
 
 
-def test_meta(capsys: pytest.CaptureFixture[str]) -> None:
-    """Test meta warning."""
-    wdl.convert(get_file("wdl_files/UmiCorrection.wdl"))
-    assert "----WARNING: SKIPPING META----" in capsys.readouterr().err
-
-
```

Comment:
No more warning?

Reply:
We don't have the warning int the new main.py